en-eng/Group-Management.md

```diff
@@ -103,6 +103,7 @@ The update configuration will will be :
 
 From that stage, you can eventually add Zigate to this group. It will make each remote controller event seen in Zigate. For that you'll have to use the configuration file and add or create a group where you have the Bulb as well as the Zigate IEEE (Zigate IEEE can be found in the utility tab under the Text Notification Widget). If you only modify the automatically created group widget, you don't have to put the Remote Controller IEEE as it has been already added in the previous stage.
 
+Additionaly, you can also enable the Left/Right commands of the Tradfri 5 buttons remote controller. When you have done 4.4 and everything is working (you are receiving the remote event to the Domoticz corresponding widget), you have to add the IEEE address of the Tradfri 5 buttons remote controller to the group you have initially created with it and for which you have added the IEEE Zigate.
```

Comment:
Additionaly, you can also use  the Left/Right buttons of the Tradfri remote controller (
![image](https://user-images.githubusercontent.com/4406440/54182600-3ec50200-44a2-11e9-8be7-c2e6b15b4deb.png) ©Ikea
. When you have done 4.4 and everything is working (you will receive the remote event in Domoticz under the corresponding widget), you have to add the IEEE address of the Tradfri remote controller to the group you have initially created with it and for which you have added the IEEE Zigate.